chapel.html.markdown

```diff
@@ -1,11 +1,11 @@
 ---
-language: chapel
-filename: learnchapel.chpl
+language: Chapel
+filename: learnChapel.chpl
```

Comment:
This should be changed back to "learnchapel.chpl" as it will break the site more then likely.